app/Http/Controllers/ContactController.php

```diff
@@ -50,10 +60,6 @@ public function sendMessage(Request $request): \Illuminate\Http\JsonResponse
      */
     protected function validator(array $data): \Illuminate\Validation\Validator
     {
-        if (! isset($data['contactDetails'])) {
-            $data['contactDetails'] = ''; // could we skip this using some feature of the validator
```

Comment:
Maybe I missed it but how did we remove the need for this workaround?

Reply:
I could've sworn I seen this work without it earlier, but I just added a test that proved me otherwise.